unit_tests/util/io.cpp

```diff
@@ -57,16 +60,18 @@ BOOST_AUTO_TEST_CASE(file_too_small)
 
         osrm::util::serializeVector(IO_TOO_SMALL_FILE, v);
 
-        std::ofstream f(IO_TOO_SMALL_FILE);
-        f.seekp(0, std::ios_base::beg);
-        std::uint64_t garbage = 0xDEADBEEFCAFEFACE;
-        f.write(reinterpret_cast<char *>(&garbage), sizeof(garbage));
+        std::fstream f(IO_TOO_SMALL_FILE);
```

Comment:
Note - this test was broken.  These changes aren't strictly part of this fingerprint change, but it did expose that this test didn't work, so I've bundled the fix in with this PR.